raven/src/main/java/com/getsentry/raven/DefaultRavenFactory.java

```diff
@@ -55,6 +55,10 @@
      */
     public static final String QUEUE_SIZE_OPTION = "raven.async.queuesize";
     /**
+     * Option for the graceful shutdown timeout of the async executor, in milliseconds.
+     */
+    public static final String SHUTDOWN_TIMEOUT = "raven.async.shutdowntimeout";
```

Comment:
This should probably be `SHUTDOWN_TIMEOUT_OPTION` for consistency with the names above.